app/renderer/js/main.js

```diff
@@ -432,6 +434,13 @@ class ServerManagerView {
 		} catch (err) {
 		}
 
+		if (ConfigUtil.getConfigItem('useServerLanguage', false)) {
+			const newLanguage = DomainUtil.getServerLanguage(this.tabs[index].webview.props.url);
+			if (newLanguage) {
```

Comment:
Can there be cases when `newLanguage` might not exist? I see that we are storing the language in preload.js.

Reply:
Yes, we're storing the language, but `newLanguage` is fetched using `DomainUtil` and I wanted to be sure that this is not `null` when changing the language to that.